packages/tools/src/tools/CursorCrosshairSync.ts

```diff
@@ -0,0 +1,581 @@
+      {
+        once: true,
+      }
+    );
+  };
+
+  //when the mouse is moved over a div, attach an event listener to this div to update the world position of the annotation when stack is scrolled
+  setMouseOverElement(element: HTMLDivElement | null): void {
+    if (element === this._mouseOverElement) return;
+    const previousElement = this._mouseOverElement;
+    this._mouseOverElement = element;
+    if (previousElement) {
+      previousElement.removeEventListener(
+        Enums.Events.VOLUME_NEW_IMAGE,
+        this.handleImageChange as EventListener
+      );
+      previousElement.removeEventListener(
+        Enums.Events.STACK_VIEWPORT_SCROLL,
+        this.handleImageChange as EventListener
+      );
+    }
+    if (element) {
+      this._mouseOverElement = element;
+      element.addEventListener(
+        Enums.Events.VOLUME_NEW_IMAGE,
+        this.handleImageChange as EventListener
+      );
+      element.addEventListener(
+        Enums.Events.STACK_VIEWPORT_SCROLL,
+        this.handleImageChange as EventListener
+      );
+    }
+  }
+
+  getActiveAnnotation(element: HTMLDivElement): null | Annotation {
+    const annotations = getAnnotations(element, this.getToolName());
+    if (annotations === undefined || annotations.length === 0) {
+      return null;
+    }
+    const targetAnnotation = annotations[0];
+    if (annotations.length > 1) {
+      const wrongAnnotations = annotations.slice(1);
+      wrongAnnotations.forEach((annotation) => {
+        if (annotation.annotationUID)
+          removeAnnotation(annotation.annotationUID, element);
+      });
+    }
+    return targetAnnotation;
+  }
+
+  //updates the position of the annotation to match the currently set world position
+  updateAnnotationPosition(
+    element: HTMLDivElement,
+    annotation: Annotation
+  ): void {
+    const worldPos = this._currentMouseWorldPosition;
+    if (!worldPos) return;
+    if (!annotation.data?.handles?.points) return;
+    annotation.data.handles.points = [[...worldPos]];
+    annotation.invalidated = true;
+
+    const viewportIdsToRender = getViewportIdsWithToolToRender(
+      element,
+      this.getToolName(),
+      false
+    );
+    const enabledElement = getEnabledElement(element);
+    if (!enabledElement) return;
+    const { renderingEngine } = enabledElement;
+    triggerAnnotationRenderForViewportIds(renderingEngine, viewportIdsToRender);
+  }
+
+  /**
+   *
+   * @param evt -  EventTypes.NormalizedMouseEventType
+   * @returns The annotation object.
+   *
+   */
+  addNewAnnotation = (
+    evt: EventTypes.MouseMoveEventType
+  ): CursorCrosshairSync => {
+    const eventDetail = evt.detail;
+    const { currentPoints, element } = eventDetail;
+    const worldPos = currentPoints.world;
+
+    const enabledElement = getEnabledElement(element);
+    if (!enabledElement) throw new Error('No enabled element found');
+    const { viewport, renderingEngine } = enabledElement;
+
+    this.isDrawing = true;
```

Comment:
I looked at your demo, the active viewport (the one controlling the cursor) has actually the pointer cursor and others have a nice + (good job on the design of that). I think it makes sense all of them have the +

Reply:
Oh I see why, since you overrode the mouseMouve hmm